ModularTegustation/tegu_items/lc13/refinery/refinery.dm

```diff
@@ -0,0 +1,77 @@
+/obj/structure/refinery
+	name = "PE Refinery"
+	desc = "A machine used to refine PE."
+	icon = 'ModularTegustation/Teguicons/refiner.dmi'
+	icon_state = "machine"
+	anchored = TRUE
+	density = TRUE
+	resistance_flags = INDESTRUCTIBLE
+	var/refine_timer = 60
+	var/timeleft
+
+	//This system has a blackjack mechanic. It gives you a number from 2 to 21 and you have to reach it with your filters.
+	//Blue - 2
+	//Green - 3
+	//Red - 6
+	//Yellow - 10
+	var/blackjack
+
+	//The state of things.
+	//FALSE is for not busy
+	//TRUE is for loaded
+	var/state = FALSE
+
+/obj/structure/refinery/attackby(obj/item/I, mob/living/user, params)
+	if(I.type == /obj/item/rawpe)
+		if(!state)
+			state = TRUE
+			to_chat(user, "<span class='notice'>You load PE into the machine.</span>")
+			qdel(I)
+			blackjack = rand(2,21)
+			timeleft = refine_timer
+			counter()
+			playsound(get_turf(src), 'sound/misc/box_deploy.ogg', 5, 0, 3)
```

Comment:
```suggestion
			playsound(get_turf(src), 'sound/misc/box_deploy.ogg', 5, 0)
```
Third num argument is extra range. We shouldn't use it everywhere, especially when it isn't something useful/important for everyone in a large area.